cli/src/main/java/org/jboss/as/cli/handlers/DeploymentHandler.java

```diff
@@ -64,25 +64,29 @@ protected void listDeployments(CommandContext ctx, boolean l) throws CommandForm
         }
 
         final StrictSizeTable table = new StrictSizeTable(names.size());
-        final List<Property> descriptions = getDeploymentDescriptions(ctx, names).asPropertyList();
-        for(Property prop : descriptions) {
-            final ModelNode step = prop.getValue();
-            if(step.hasDefined(Util.RESULT)) {
-                final ModelNode result = step.get(Util.RESULT);
-                table.addCell(Util.NAME, result.get(Util.NAME).asString());
-                table.addCell(Util.RUNTIME_NAME, result.get(Util.RUNTIME_NAME).asString());
-                if(result.has(Util.ENABLED)) {
-                    table.addCell(Util.ENABLED, result.get(Util.ENABLED).asString());
+        try {
+            final List<Property> descriptions = getDeploymentDescriptions(ctx, names).asPropertyList();
+            for(Property prop : descriptions) {
+                final ModelNode step = prop.getValue();
+                if(step.hasDefined(Util.RESULT)) {
+                    final ModelNode result = step.get(Util.RESULT);
+                    table.addCell(Util.NAME, result.get(Util.NAME).asString());
+                    table.addCell(Util.RUNTIME_NAME, result.get(Util.RUNTIME_NAME).asString());
+                    if(result.has(Util.ENABLED)) {
+                        table.addCell(Util.ENABLED, result.get(Util.ENABLED).asString());
+                    }
+                    if(result.has(Util.STATUS)) {
+                        table.addCell(Util.STATUS, result.get(Util.STATUS).asString());
+                    }
                 }
-                if(result.has(Util.STATUS)) {
-                    table.addCell(Util.STATUS, result.get(Util.STATUS).asString());
+                if(!table.isAtLastRow()) {
+                    table.nextRow();
                 }
             }
-            if(!table.isAtLastRow()) {
-                table.nextRow();
-            }
+            ctx.printLine(table.toString());
+        } catch (Exception e) {
```

Comment:
I think this is not required in normal scenario. However, I did not have absolute confidence that I can totally remove throwing the CommandFormatException from there. That's why I added such `try ~ catch (Exception e) {...}` block to throw the CommandFormatException for safety's sake.

Is it safe to simply replace `throw new CommandFormatException(table.toString())` with `ctx.printLine(table.toString())` (without adding such `try-catch` block)?